### PR TITLE
Update with Eliom dev version.

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: src/how-to-do-links-to-other-pages.wiki [hunk 1]
```diff
@@ -1,28 +1,24 @@
 =How to do links to other pages?=
 
-Examples:
+If you want to make a link to an other page in your code, you can use the Eliom_content.Html.D.a function which is the functional representation for the <a> HTML tag. Supposing Eliom_content.Html.D is opened, here some examples with the previous created services:
```
Comment (vasilisp) [inline src/how-to-do-links-to-other-pages.wiki]: Please use monospace where appropriate, e.g., `{{{Eliom_content.Html.D.a}}}` and `{{{<a>}}}`.

File: src/how-to-do-links-to-other-pages.wiki [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+    ~id:(Eliom_service.External ("https://ocsigen.org", [""]))
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
     ())
-  [pcdata "OCaml on wikipedia"]
-  ["OCaml"];
-
-(* Low level interface for links: *)
-Html5.F.Raw.a
-  ~a:[a_href (Xml.uri_of_string "http://en.wikipedia.org/wiki/OCaml")]
-  [pcdata "OCaml on wikipedia"]
```
Comment (vasilisp) [inline src/how-to-do-links-to-other-pages.wiki]: Why has this been removed?

**Comment by dannywillems** [comment]
I didn't saw it.

File: src/how-to-do-links-to-other-pages.wiki [hunk 3]
```diff
@@ -1,28 +1,63 @@
 =How to do links to other pages?=
 
-Examples:
+Sometimes, you need to create links to other websites. For Eliom, these links are called «external services». These external services are services which are not registered and generated by your application (for example a link to https://ocsigen.org).
```
Comment (vasilisp) [inline src/how-to-do-links-to-other-pages.wiki]: I would much prefer the tutorial to start with standard internal services, these come up more frequently. And then you can say that it's easy to use the same mechanism for external links, with examples like the ones you have already.

File: src/how-to-do-links-to-other-pages.wiki [hunk 4]
```diff
@@ -1,28 +1,63 @@
...
+This code create an external service for https://ocsigen.org. The function Eliom_service.External takes a tuple (a, b) where a is the URL to the page as a string and the second is an array representing the path. Another example for the page https://ocsigen.org/tuto/manual:
+<<code language="ocaml"|
+let ocsigen_tuto_manual_service =
+  Eliom_service.create
+    ~id:(Eliom_service.External ("https://ocsigen.org", ["tuto", "manual"]))
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
+    ()
+>>
+
+By changing the parameter ~meth, you can also send GET and POST parameters, like for an internal service (ie registered by your application). For example, if http://example.com needs three GET parameters of type int, int and string, you create an external service to the site with
```
Comment (vasilisp) [inline src/how-to-do-links-to-other-pages.wiki]: That's an explanation of how services work. As such, it is beyond the point in a howto about links (which can have no POST parameters).

**Pull request closed without merging**